gluten-ut/spark33/src/test/scala/org/apache/spark/sql/GlutenFileScanSuite.scala

```diff
@@ -16,4 +16,15 @@
  */
 package org.apache.spark.sql
 
-class GlutenFileScanSuite extends FileScanSuite with GlutenSQLTestsBaseTrait {}
+import io.glutenproject.execution.FileSourceScanExecTransformer
+import org.apache.spark.sql.GlutenTestConstants.GLUTEN_TEST
+
+class GlutenFileScanSuite extends FileScanSuite with GlutenSQLTestsBaseTrait {
+  test(GLUTEN_TEST + "Filter on short column type pushed to native") {
+    withTable("abc") {
+      spark.sql("create table abc (a short, b int) using parquet")
+      val plan = spark.sql("select * from abc where a = 1").queryExecution.executedPlan
+      assert(plan.find(_.isInstanceOf[FileSourceScanExecTransformer]).isDefined)
+    }
+  }
+}
```

Comment:
does this issue only happens in spark33? could you move this ut to backend-velox module?

Reply:
Thanks for suggestion. Moved test to backend-velox

Reply:
I'd expect move this test into backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala, like below
https://github.com/oap-project/gluten/blob/146b82f66e54166f01c3423b1b2b31a894a3c950/backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala#L721-L735

Reply:
Moved the test to this file.

Reply:
thanks, and could we remove this test? since TestOperator.scala already has same test logic and that can be tested for all spark version, but this test could only be run in spark33.

Is any particular reason to keep this test?

Reply:
Oh my bad, forgot to remove this. Thanks